context/brand-context/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@springernature/brand-context",
-  "version": "10.1.1",
+  "version": "10.1.2",
```

Comment:
Looks like you've bumped a patch but meant a major

Reply:
Sorry. I should have removed that text from the PR. I discussed this with Alex and he suggested it should be a patch.

Reply:
Ah I see. In the case the `History` entry is incorrect 😄